Ktisis/Ktisis.csproj

```diff
@@ -24,6 +24,10 @@
   <PropertyGroup>
     <DalamudLibPath>$(appdata)\XIVLauncher\addon\Hooks\dev\</DalamudLibPath>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT' OR '$(CI)' == 'true'">
```

Comment:
I know I'm a bit late on this, but shouldn't this condition just be `$(CI) == 'true'`?

Reply:
It's a holdover from another project that allows me to use a VSCode devcontainer and set the same environment variable for Dalamud location, doesn't need to stay if unwanted